doc/source/release-notes/Percona-XtraDB-Cluster-5.7.20-29.24.rst

```diff
@@ -0,0 +1,121 @@
+.. rn:: 5.7.20-29.24
+
+=====================================
+Percona XtraDB Cluster 5.7.20-29.24
+=====================================
+
+Percona is glad to announce the release of
+Percona XtraDB Cluster 5.7.20-29.24 on January 26, 2018.
+Binaries are available from the `downloads section
+<http://www.percona.com/downloads/Percona-XtraDB-Cluster-57/>`_
+or from our :ref:`software repositories <install>`.
+
+.. note:: Due to new package dependency,
+   Ubuntu/Debian users should use ``apt-get dist-upgrade``, ``apt upgrade``,
+   or ``apt-get install percona-xtradb-cluster-57`` to upgrade.
+
+Percona XtraDB Cluster 5.7.20-29.24 is now the current release,
+based on the following:
+
+* `Percona Server 5.7.20-18 <https://www.percona.com/doc/percona-server/5.7/release-notes/Percona-Server-5.7.20-18.html>`_
+
+* Galera Replication library 3.22
+
+* Galera/Codership WSREP API Release 5.7.20
+
+All Percona software is open-source and free.
+
+NEW FEATURES:
+=============
+
+* Ubuntu 17.10 Artful Aardvark is now supported.
+
+* Freezing gcache purge was implemented to facilitate node
+  joining through IST, avoiding time consuming SST process.
+
+* :jirabug:`PXC-822`: an usability improvement was made in timeout error
+  messages making them to name the configuration variable which caused this
+  timeout.
```

Comment:
rephrasing

-----
a usability improvement was made to timeout error messages, the name of the configuration variable which caused the timeout was added to the message.